phpcs.xml.dist

```diff
@@ -6,8 +6,10 @@
 
     <config name="installed_paths" value="../../slevomat/coding-standard"/>
 
-    <!--The complete PSR-2 ruleset-->
-    <rule ref="PSR2"/>
+    <!-- The complete PSR-12 ruleset (excluding a rule that requires PHP >= 7.1 -->
```

Comment:
Missing ")" but maybe the comment for constant visibility should be separated and moved just before the relevant `<exclude>`...

Reply:
Done.